### DISCUSSION TITLE
Support playback while the Graphic EQ window is open

### DISCUSSION THREAD

**Discussion by gy-mate** [audacity/audacity#5650] [Q&A] [answered]
### Your idea

Support playback of clips or selection while the Graphic EQ window is open in the foreground. Also, if any of the EQ sliders' state is altered, the change should be applied realtime during playback.

### Problem to be solved

It is only possible to play the EQ-adjusted clip with the 'Preview' button, but
- it opens a new window,
- EQ sliders cannot be toggled during playback,
- it only plays the clip for a few seconds.

These make it much more difficult to find the correct EQ settings.

### Prior art

Adobe Audition allows endless realtime EQ playback and adjustments.

**Comment by LWinterberg** [comment, marked as answer]
We have realtime effects when you use the effects button in the tracks themselves instead of the effects menu. Graphic EQ isn't supported by that system yet, but you can download some plugins from https://plugins.audacityteam.org/realtime-effects/equalizers which all work in that mode.

**Comment by Paul-Licameli** [comment]
Trust us it’s been much thunk of

**Comment by petersampsonaudacity** [comment]
> * it only plays the clip for a few seconds.

@gy-mate 

You can change the duration in preferences.

Go to **Playback** preferences and change the value for **Effects Preview > Length**

Peter.

**Reply by gy-mate** [reply]
Thanks for the tip! :)
